EXT_BeamPhysics.md

```diff
@@ -142,6 +142,10 @@ The following records store data on a particle-by-particle basis.
   - description: Base momentum from which `momentum` is measured. That is, True momentum = `momentum + momentumOffset`. Assumed zero if not present.
   - components: (`x`, `y`, `z`).
 
+-`particleID`
```

Comment:
a globally unique particle `id` is already available in the base standard to hook into. Would this work for you?
https://github.com/openPMD/openPMD-standard/blob/upcoming-2.0.0/STANDARD.md#records-for-each-particle-species

Reply:
@ChristopherMayes  Does the base standard "id" work for you? Notice that the description is

"description: a globally-unique identifying integer for each particle, that can be used to, e.g., track particles. This identifying integer should be truly unique within the simulation; in particular, even among different particle species, two particles should not have the same id. Also, when a particle exits the simulation box, its identifying integer should not be reassigned to a new particle."

@ax3l  The base standard needs to be modified to not use “Particle Species”. The beam physics extension uses the name “Particle Group”. Also the standard should not be mandating what type of integer should be used as it is for "id".

Reply:
@ax3l I should note that my code does not use ID numbers so I am fine with what everyone else wants.

Reply:
@DavidSagan `id` is fine for me.

Reply:
@ax3l Take a look at the change and tell me if this is OK.

Reply:
Looks great, good idea to mention the reference again here.